src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -188,5 +189,24 @@ public override string ToString()
         {
             return DisplayText;
         }
+
+        private List<string> tags = null;
+        public IEnumerable<string> Tags
+        {
+            get
+            {
+                return tags ?? SpecializedCollections.EmptyEnumerable<string>();
+            }
+        }
+
+        public void AddTag(string tag)
```

Comment:
I wanted to avoid cluttering the constructor with yet an other mysterious parameter :smile: